.vscode/launch.json

```diff
@@ -44,8 +44,8 @@
         {
             "cwd": "${workspaceRoot}",
             // TODO: find better way to get latest build filename
-            "executable": "./build/src/pinetime-app-1.3.0.out",
-            "name": "Debug OpenOCD ST-LINK pinetime-app-1.3.0.out",
+            "executable": "./build/src/pinetime-app-1.11.0.out",
```

Comment:
If the vscode launch script must be improved, let's do this in another PR.

Reply:
I reverted the name but had to keep the change for the executable in order to run the debugger.

Reply:
You don't need to commit the change.

Reply:
Change is reverted.